app/discussion/client/views/DiscussionTabbar.js

```diff
@@ -24,24 +28,23 @@ Template.discussionsTabbar.helpers({
 
 Template.discussionsTabbar.onCreated(function() {
 	this.rid = this.data.rid;
-	this.cursor = DiscussionOfRoom.find({
-		rid: this.rid,
-	}, {
-		sort: {
-			ts: -1,
-		},
-	});
 	this.hasMore = new ReactiveVar(true);
 	this.limit = new ReactiveVar(50);
-	return this.autorun(() => {
+	this.cursor = new ReactiveVar([]);
+	this.autorun(() => {
 		const data = Template.currentData();
-		return this.subscribe('discussionsOfRoom', data.rid, this.limit.get(), () => {
-			if (DiscussionOfRoom.find({
+		const handle = Meteor.subscribe('discussionsOfRoom', data.rid, this.limit.get());
```

Comment:
if you are using `Meteor.subscribe` you should include a `sub.stop()` `this.subscribe` make the stop on `onDestroy` to you.

Reply:
updated